app/views/cycles/plugin_relations/petition.html.slim

```diff
@@ -43,15 +43,19 @@ section#petition-index
             | assinaturas para a meta de
             strong=" #{@petition.signatures_required}"
 
-        .row.small-gap.remaining-days.bottom-info-container
-          .col-xs-12
-            .pull-left
-              i.icon-clock style="color: #{@cycle.color}"
-            div.medium-padded
-              strong= "#{@phase.remaining_days} dias "
-              | restantes para o encerramento
-            div.medium-padded
-              small= "As colaborações podem ser feitas até o dia #{@phase.final_date.strftime("%d/%m/%Y")}" 
+        - if @phase.in_progress?
+          .row.small-gap.remaining-days.bottom-info-container
+            .col-xs-12
+              .pull-left
+                i.icon-clock style="color: #{@cycle.color}"
+              div.medium-padded
+                - if @phase.remaining_days > 0
```

Comment:
You could add this to a a view helper and not to worry about doing this twice in this template.